src/main/java/com/cloudbees/jenkins/plugins/bitbucket/client/BitbucketCloudApiClient.java

```diff
@@ -504,6 +518,37 @@ public BitbucketTeam getTeam() throws IOException, InterruptedException {
         return getRepositories(null);
     }
 
+    private synchronized HttpClient getHttpClient() {
+        if (this.client == null) {
+            HttpClient client = new HttpClient(connectionManager);
```

Comment:
Why does a new HttpClient need to be instantiated? There's already one instantiated in the constructor of BitbucketCloudApiClient.